src/main.cpp

```diff
@@ -61,7 +61,23 @@ int main(int argc, char * argv[]) {
         args.printUsage();
         return 0;
     }
-
+    if(args.getSettingsPathSet()) {
+        QDir settingsPath(args.getSettingsPath());
+        qDebug() << "Settings Path Provided: " << settingsPath.absolutePath();
+        if(!args.getSettingsPathStatus()) {
+            qDebug() << "The provided settings path does not have the proper permissions";
+            QApplication app(argc, argv);
+            QMessageBox::critical(0, QObject::tr("Cannot access this path"),
+                                     QObject::tr("The Settings Path: %1 that is "
+                                        "provided does not have the "
+                                        "proper permission to access its "
```

Comment:
Ah, I see. You schould aso check the default path. But you may change the message according to the getSettingsPathSet()